### PR TITLE
Clarify default tenant/app name

### PR DESCRIPTION
**Pull request by kkraune**
ref user feedback in https://github.com/vespa-engine/vespa/issues/26493

### CODE REVIEW

File: en/reference/deploy-rest-api-v2.html [hunk 1]
```diff
@@ -7,10 +7,14 @@
 ---
 
 <p>
-  This is the API specification and some examples for the  HTTP Deploy API that can be used to deploy applications in Vespa.
-  The API supports multiple tenants  and applications. Each tenant may have multiple applications deployed, all of which
-  can be deployed using this API.
-</p><p>
+  This is the API specification and some examples for the HTTP Deploy API that can be used to deploy applications in Vespa.
+  The API supports multiple tenants and applications.
+  Each tenant may have multiple applications deployed, all of which can be deployed using this API.
```
Comment (bratseth) [inline en/reference/deploy-rest-api-v2.html:12]: We only support *one* application.

**Comment by kkraune** [comment]
ptal, no room for confusion now

**Comment by kkraune** [comment]
fyi @hmusum

**Review by hmusum — APPROVED** [review]
Looks good to me

**Review by bratseth — APPROVED** [review]

**Pull request merged**